lib/main_common.pm

```diff
@@ -2060,9 +2060,10 @@ sub load_applicationstests {
 }
 
 sub load_security_console_prepare {
-    loadtest "console/system_prepare";
     loadtest "console/consoletest_setup";
-    loadtest "console/hostname" if get_var("SECURITY_TEST") !~ /crypt_krb5/;
+    loadtest "security/test_repo_setup" if
```

Comment:
Why not in one line?

Reply:
It is 80 character's convention to make it easy to read, but I know it is not applicable to Perl. Modified!